tests/unit/stubs/misc.ts

```diff
@@ -0,0 +1,48 @@
+import sinon from "sinon"
+import check from "express-validator/check";
+import { hash } from "../../../src/util/hash";
+
+export const validatorStubs = () => {
+    return {
+        validationResult: sinon.stub(check, "validationResult"),
+
+        restore() {
+            this.validationResult.restore();
+        }
+    };
+};
+
+export const emptyValidationError = () => {
+    return {
+        isEmpty() {
+            return true;
+        }
+    }
+};
+
+export const validationErrorWithMessage = (errorMsg: { status: number; message: string }) => {
+    return {
+        isEmpty() {
+            return false;
+        },
+        formatWith() {
+            return {
+                array() {
+                    return [errorMsg];
+                }
+            };
+        }
+    };
+}
+
+export const problemUtilStubs = () => {
+    return {
+        hash: {
+            calculateProblemHash: sinon.stub(hash, "calculateProblemHash")
+        },
+
+        restore() {
+            this.hash.calculateProblemHash.restore();
+        }
+    }
+}
```

Comment:
This is not super consistent with what we have in the Users Microservice - maybe a discussion about how we group stubs is beneficial?